docs/usage/configuration.md

```diff
@@ -55,16 +55,18 @@ The plugin uses the `ci`, `location`, and `apiKey` parameters to determine the c
 
 ### Accounting for a personal energy mix
 
-The `ciMarket` parameter can be used to provide a custom value to account for differences to your regional average. This can occur due to:
-- A different market share through a contract with your energy provider, guaranteeing to provide a certain percentage of electricity from renewable sources
-- Direct contributions to the used electricity (e.g. via owned solar panels)
+The `ciMarket` parameter can be used to provide a custom value to account for differences to your regional average. This can occur due to:  
 
-You can calculate an approximation of your CI via the average of the [emission factors](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Default-emission-factors) weighted by their respective share in your mix.
+-  A different market share through a contract with your energy provider, guaranteeing to provide a certain percentage of electricity from renewable sources  
+-  Direct contributions to the used electricity (e.g. via owned solar panels)
+
+You can calculate an approximation of your regional CI via the average of the [emission factors](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Default-emission-factors) weighted by their respective share in your mix.
```

Comment:
`approximation of your regional CI`: I find the term `regional` confusing in this context, wouldn't this then be more the `personal` or `market-based` CI? In contrast to the `location-based` CI?